gitlab/client.py

```diff
@@ -698,7 +714,8 @@ def http_request(
                 "retry_transient_errors", self.retry_transient_errors
             )
```

Comment:
Could we have this block removed as you've now added it above? I think it might just be a rebase issue.

Reply:
Done in https://github.com/python-gitlab/python-gitlab/pull/1904/commits/5cbbf26e6f6f3ce4e59cba735050e3b7f9328388.